### ISSUE TITLE
Typo in documentation of Flag

### ISSUE THREAD

**Issue by PanderMusubi** [wtforms/wtforms#724]
The documentation says that for non-existing flags the value should be `None`, see https://github.com/wtforms/wtforms/blob/master/src/wtforms/fields/core.py#L407

But the example code returns `False`, see https://github.com/wtforms/wtforms/blame/master/docs/fields.rst#L562

Is this a typo? What should be corrected? The example, the example or the documentation?

**Comment by azmeuk** [comment]
It seems to me there is no issue here:

```python
>>> from wtforms.fields.core import Flags
>>> flags = Flags()
>>> flags.required = True
>>> 'required' in flags
True
>>> 'nonexistent' in flags
False
>>> flags.required
True
>>> flags.nonexistent
```

**Comment by PanderMusubi** [comment]
Sorry should https://github.com/wtforms/wtforms/blame/master/docs/fields.rst#L564 return nothing instead of `False`?

For the rest it indeed checks out

    >>> flags.nonexistent == None
    True

**Comment by azmeuk** [comment]
> Sorry should https://github.com/wtforms/wtforms/blame/master/docs/fields.rst#L564 return nothing instead of `False`?

Yep! Do you want to submit a patch?

**Comment by PanderMusubi** [comment]
I'm active in Bootstrap Flask and will leave this one for you.

**Comment by PanderMusubi** [comment]
Testing with `None` is more insightful as return of nothing is hard to document in example Python code.

**Comment by azmeuk** [comment]
Fixed by 0c7aee2
